### PR TITLE
Revert "Splitting seralizers into read and write ones"

### PR DESCRIPTION
**Pull request by fao89**
Reverts pulp/pulpcore#707

### CODE REVIEW

**Comment by fao89** [comment]
Should we file an issue and add a changelog?
@bmbouter @daviddavis @dkliban

**Comment by mdellweg** [comment]
Is reverting not introducing the original problem?

**Comment by dkliban** [comment]
> Is reverting not introducing the original problem?

Yes, we decided to abandon this approach to solving the problem. Instead we are going to switch to using OpenAPI v3 for pulpcore 3.6. We can achieve this by switching to drf-spectacular. drf-spectacular provides a guide for migrating from drf-yasg. 

https://drf-spectacular.readthedocs.io/en/latest/
https://drf-spectacular.readthedocs.io/en/latest/drf_yasg.html

File: CHANGES/7087.removal [hunk 1]
```diff
@@ -0,0 +1 @@
+Revert: Fixed a bug that prevented write_only fields from being present in the API docs and bindings
```
Comment (dkliban) [inline CHANGES/7087.removal:1]: Removes the Write models from the OpenAPI schema. This also brings back the models that were accidentally removed from the OpenAPI schema in 3.4.0 release.

**Pull request merged**